src/KintoID.sol

```diff
@@ -343,7 +357,24 @@ contract KintoID is
      * @return true if the account has KYC token.
      */
     function isKYC(address _account) external view override returns (bool) {
-        return balanceOf(_account) > 0 && isSanctionsSafe(_account);
+        if (balanceOf(_account) == 0) {
+            // If the account doesn't have a KYC token
```

Comment:
this comment is not correct

Reply:
I believe we should just return false in this case, if there is no NFT it should be false